app/controllers/propertyLinking/ChooseEvidenceController.scala

```diff
@@ -57,7 +57,13 @@ class ChooseEvidenceController @Inject()(
       _        <- businessRatesAttachmentService.persistSessionData(request.ses)
       backLink <- backlink(request.ses)
     } yield {
-      Ok(chooseEvidenceView(form, request.ses.clientDetails, Some(backLink)))
+      Ok(
+        chooseEvidenceView(
+          form,
+          request.ses.clientDetails,
+          Some(backLink),
+          request.ses.address,
+          request.ses.localAuthorityReference))
```

Comment:
you don't really have to do this as the request is available in the view,

So this would be neater I think, and in the view just access the fields you need via the request
```
chooseEvidenceView(form,  Some(backLink))
```